packages/cdk8s-cli/src/cli/index.ts

```diff
@@ -3,16 +3,22 @@ import * as yargs from 'yargs';
 import { upgradeAvailable } from '../upgrades';
 
 async function main() {
-  const versions = upgradeAvailable();
-  if (versions) {
-    console.error('------------------------------------------------------------------------------------------------');
-    console.error(colors.yellow(`A new version ${versions.latest} of cdk8s-cli is available (current ${versions.current}).`));
-    console.error(colors.yellow('Run "npm install -g cdk8s-cli" to install the latest version on your system.'));
-    console.error(colors.yellow('For additional installation methods, see https://cdk8s.io/docs/latest/getting-started'));
-    console.error('------------------------------------------------------------------------------------------------');
-  }
-
   const ya = yargs
+    .option("no-check-upgrade", {type: "boolean", desc: "Do not check for cdk8s-cli upgrade"})
```

Comment:
Please consider using `check-upgrade` instead of `no-check-upgrade` and set default to true. This way you can still do `--no-check-upgrade` and also use CDK8S_CHECK_UPGRADE=false.

Reply:
Good idea. I'll do that 👍

Reply:
Just updated the PR to implement those changes.